tests/tian_water_mass_conservation.prm

```diff
@@ -34,8 +34,8 @@
 ## is 0.03261, verifying that we are conserving the mass of water. 
 
 include $ASPECT_SOURCE_DIR/tests/tian_peridotite.prm
-set End time                               = 1e2
-set Maximum time step                      = 1e2
+set End time                               = 1e3
+set Maximum time step                      = 1e3
```

Comment:
why this change? is it so that now the fluid reaction is split into several steps? shouldnt then the results of the tests change?